CautionaryAlertsListener.Tests/CreateCautionaryAlertFixture.cs

```diff
@@ -0,0 +1,41 @@
+using AutoFixture;
+using Hackney.Shared.CautionaryAlerts.Boundary.Request;
+using Hackney.Shared.CautionaryAlerts.Infrastructure;
+using System;
+
+namespace CautionaryAlertsListener.Tests
+{
+    public static class CreateCautionaryAlertFixture
+    {
+        public static CreateCautionaryAlert GenerateValidCreateCautionaryAlertFixture(string defaultString, Fixture fixture, Guid? mmhId = null, string propertyReference = null)
```

Comment:
Is this being used anywhere?

Reply:
![image](https://user-images.githubusercontent.com/10539291/207357900-5040c464-cc37-410d-8593-0e8d8b6cbba5.png)